source/wp-content/themes/wporg-news-2021/sass/components/_categories.scss

```diff
@@ -45,21 +45,63 @@ body.category-month-in-wordpress {
 		color: var(--wp--preset--color--off-white-2);
 	}
 
-	.wp-block-post-date a {
-		color: var(--wp--preset--color--off-white-2);
-		text-transform: uppercase;
-	}
-
-	.last-in-year::after {
-		content: "-";
-		display: inline-block;
-		line-height: calc(var(--wp--custom--margin--vertical) * 6);
-		width: 100%;
-		background-image: url(images/brush-stroke-short-blue-4.svg);
-		background-position: center;
-		background-repeat: no-repeat;
-		//background-size: fill;
-		color: var(--wp--preset--color--blue-1);
+	.wp-block-post-template {
+		margin-top: calc(var(--wp--custom--margin--vertical) * 2);
+		margin-bottom: calc(var(--wp--custom--margin--vertical) * 2);
+	}
+
+	.wp-block-post {
+		text-align: center;
+		margin-bottom: calc(var(--wp--custom--margin--vertical) * 1.333);
+		@include break-small-only() {
+			width: 50%;
+		}
+	}
+
+	.wp-block-post-date {
+		font-size: var(--wp--preset--font-size--normal);
+		a {
+			color: var(--wp--preset--color--off-white-2);
+			text-transform: uppercase;
+		}
+	}
+
+	.last-in-year {
+		/* We are removing this item from the grid so we can make the ::after element work as a child of the grid*/
+		display: contents;
+		position: relative;
+		break-after: always;
+		&::after {
+			content: "-";
+			line-height: calc(var(--wp--custom--margin--vertical) * 6);
+			width: 100%;
+			mask-image: url(images/brush-stroke-big.svg);
+			mask-position: center;
+			mask-repeat: no-repeat;
+			mask-size: 100% auto;
+			background-color: var(--wp--preset--color--blue-1);
+			transform: translateY(-25%);
+			@include break-small() {
+				mask-size: 280px auto;
+			}
+		}
+		article {
+			/* We are making the article look like it's a child of the grid container */
+			width: calc(33.33333% - .83333em);
+			margin-bottom: calc(var(--wp--custom--margin--vertical) * 2.5);
+			@include break-small-only() {
+				width: 50%;
+			}
+		}
+	}
+
+	h2 time {
+		font-size: min(max(50px, 5vw), 120px);
+		line-height: 1;
+		a:hover,
+		a:focus {
+			text-decoration: none;
```

Comment:
This is removing any hover effect - if we don't want an underline, can there be another visual indicator? cc @beafialho

Reply:
These posts are meant to have an [underline and color change on hover](https://www.figma.com/file/O9I8Jl7kaUMTX39LlRGbGO/News-Page?node-id=4910%3A25499). Is this what you're referring to?

<img width="1486" alt="Captura de ecrã 2022-01-14, às 16 37 14" src="https://user-images.githubusercontent.com/65220155/149551784-8ce6760f-fbca-453c-b2ef-239ef061a0ae.png">

Reply:
Fixed now!

Reply:
Also passed the linting too, thanks for the command!

Reply:
Looks great (I'm able to test locally now!). One question @MaggieCabrera, is the H2 font size used in the month's names `50px`? If it is, could we set this category's titles to be `120px`? Asking because I'm seeing post titles in a smaller size than the intended across the whole site and wanted to make sure that's not intentional.

Reply:
@beafialho I think what you are seeing is a bug with the names of the variables being incorrect for the font sizes, and it's unrelated to this PR, I think Danny is going to fix that. In this particular case I was using a dynamic value for the font size that goes from 50px on mobile to 120px on desktop, I haven't seen a variable for that, maybe I should create one in case the same is reused somewhere else.

Reply:
I see what you mean, that should be fine then. Thanks!